Add support for PageSerializationMode.VIA_DTO (PagedModel) in PageJacksonModule

bruce-stewart opened the pull request:
This pull request addresses issue #1144

OlgaMaciaszek reviewed (changes requested):
Hello @bruce-stewart, thanks for submitting the PR. Looks good. Please update the date in the license comments on top of all the java classes you've changed to `-2025`. Also, please add the `@author` tag with your full name and surname to the javadocs of the classes you've modified.

We now require the contributors to adhere to the DCO by signing-off their commits. Please follow these steps to amend your commits: https://github.com/spring-cloud/spring-cloud-openfeign/pull/1189/checks?check_run_id=41357021058.

bruce-stewart commented:
@OlgaMaciaszek Apologies for the clearly chaotic commit history on this - I have very little experience with contributing to Github projects, so my attempts at re-basing seem to be creating a lot of noise! I have updated the Copyright date and added my name as an author. I've been trying to clean up the sign-off commit comment, but am not having much success.  Please let me know if there is something more that I should fix.

OlgaMaciaszek commented:
Hi @bruce-stewart, unfortunately, the commit sign-off will be required (`git commit --signoff`). Usually, these steps work: https://github.com/spring-cloud/spring-cloud-openfeign/pull/1189/checks?check_run_id=41855140861. If there's an issue with it, since there not many changes, you could just create a new branch, copy in the changes and commit them with sign-off. Then push it and create a new pull request.

bruce-stewart commented:
Thanks @OlgaMaciaszek I'll do a new pull request

bruce-stewart commented:
I've created a new PR #1193
